src/dpr/components/columns/utils.ts

```diff
@@ -7,6 +7,7 @@ export default {
         text: f.display,
         value: f.name,
         disabled: f.mandatory,
+        visible: f.visible,
```

Comment:
Is this new property used? I can't see it in the view or elsewhere.

Reply:
src/dpr/types/ReportQuery.ts

Reply:
reallly small change to get this to work. Way more code to fix the tests